src/main/java/lotto/model/Lotto.java

```diff
@@ -5,12 +5,30 @@
 public class Lotto {
     private List<Integer> numbers;
 
-
     public Lotto(final List<Integer> numbers) {
         this.numbers = numbers;
     }
 
     public List<Integer> getNumbers() {
         return numbers;
     }
+
+  public Result match(Lotto winningLotto, int bonus, Lotto myLotto) {
+        int count = safeLongToInt(winningLotto.getNumbers().stream()
+                .filter(number -> myLotto.getNumbers().contains(number))
+                .count());
+
+        boolean isBonus = winningLotto.getNumbers().stream()
+                .anyMatch(number -> myLotto.getNumbers().contains(bonus));
```

Comment:
winningLotto가 필요한가요?
아래와 같이 바꿔볼 수 있을까요?
```suggestion
        boolean isBonus = numbers.contains(bonus);
```